aiida_common_workflows/workflows/relax/abinit/generator.py

```diff
@@ -51,6 +52,11 @@ def define(cls, spec):
         spec.inputs['electronic_type'].valid_type = ChoiceType(
             (ElectronicType.METAL, ElectronicType.INSULATOR, ElectronicType.UNKNOWN)
         )
+        spec.inputs['protocol'].valid_type = ChoiceType((
+            'fast', 'moderate', 'precise', 'verification-PBE-v1-PAW-11', 'verification-PBE-v1-NC-04',
+            'verification-PBE-v1-NC-05b1', 'verification-PBE-v1-PD-experiment-ne-high',
```

Comment:
I believe these new protocols are not defined in the  `protocol.yml`, they should be removed here. The changes for the Ne element should be integrated in the respective protocol if accepted, or discarded entirely.